setup.cfg

```diff
@@ -1,3 +1,10 @@
+[build-system]
```

Comment:
thanks for getting a solution. I suppose now it's finally working? Please include a reference to https://setuptools.readthedocs.io/en/latest/setuptools.html#distributing-extensions-compiled-with-cython.

Reply:
Ok ; I'm not full time on it; I still need to test it. I'll let you know when this is done.